src/usdb_syncer/notes_parser.py

```diff
@@ -456,6 +461,7 @@ def fix(self) -> None:
         self.fix_low_bpm()
         self.fix_line_breaks()
         self.fix_touching_notes()
+        self.fix_pitch_values()
 
     def minimum_song_length(self) -> str:
```

Comment:
I think it's considered best practice not to pass in more than the minimum required arguments into a function. In the case of a method, this would mean passing in the 'smallest self'.
In more understandable terms: You can move this method into `Tracks` since `self` is only ever used to refer to `self.notes`.

Reply:
As always, thanks for the valuable feedback!